book/src/user/supported-platforms.md

```diff
@@ -0,0 +1,48 @@
+# Platform Support
+
+Support for different platforms are organized into three tiers, each with a
+different set of guarantees. For more information on the policies for platforms
+at each tier, see the [Platform Tier Policy](platform-tier-policy.md).
+
+Platforms are identified by their "target triple" which is a string composed by
+`<machine>-<vendor>-<operatingsystem>`.
+
+## Tier 1
+
+Tier 1 platforms can be thought of as "guaranteed to work". The Zebra project
+builds official binary releases for each tier 1 platform, and automated testing
+ensures that each tier 1 platform builds and passes tests after each change.
+
+For the full requirements, see [Tier 1 platform policy](platform-tier-policy.md#tier-1-platform-policy) in the Platform Tier Policy.
+
+| platform | os | notes | artifacts
+| -------|-------|-------|-------
+| `x86_64-unknown-linux-gnu` | Debian 11.4 | 64-bit (rust 1.62.0) | Docker
```

Comment:
I think it might be easier to read if we make the Rust compiler version into a separate column

Reply:
Fixed by [fix(docs): make rust version a separate column](https://github.com/ZcashFoundation/zebra/pull/4773/commits/f67eedccd648e74fbb4c286c3c189cc8f04b0e4b)